theme/js/components/discussions/threads.vue

```diff
@@ -1,66 +1,71 @@
 <template>
   <section class="discussions-wrapper" ref="top" key="top">
+    <div class="fr-grid-row">
+      <div class="fr-col">
+        <h2 id="community-discussions" class="fr-h2">{{ title }} <sup>{{totalResults}}</sup></h2>
+        <slot></slot>
+      </div>
+      <div class="fr-col-12 fr-col-sm-5 fr-col-md-4 fr-col-lg-3 fr-grid-row fr-grid-row--bottom flex-direction-column justify-between" v-if="!threadFromURL">
+        <ThreadsCreateButton class="fr-col--bottom" :onClick="startThreadWithoutScroll"/>
+        <div class="fr-mt-5v">
+          <select
+          name="sortBy"
+          id="sortBy"
+          @change="changeSort(currentSort)"
+          v-model="currentSort"
+          class="fr-select bg-beige fr-select--no-border"
+        >
+          <option
+            v-for="sort in sorts"
+            :value="sort"
+            :selected="sort === currentSort"
+          >
+            {{ sort.name }}
+          </option>
+        </select>
+        </div>
+      </div>
+    </div>
     <transition mode="out-in">
-      <div v-if="loading" key="loader">
+      <template v-if="loading" key="loader">
         <Loader class="mt-md" />
-      </div>
-      <div v-else>
+      </template>
+      <template v-else>
         <div v-if="threadFromURL">
-          <div class="well well-secondary-green-300">
-            <div class="row-inline justify-between">
+          <div class="fr-px-3w well well-secondary-green-300">
```

Comment:
Do we want to try a [DSFR callout](https://gouvfr.atlassian.net/wiki/spaces/DB/pages/222331196/Mise+en+avant+-+Call+out) here ? The main difference is the button which is only a "close" for us (and only a `x` before) but is a real button on the DSFR callout.

Reply:
Yes please, I'm curious! :pray:

Reply:
This is a screenshot with bot the callout and the well to see the differences. I use green to match the previous design but I don't really know if it's necessary.
If we want a smaller callout, we can remove the button but the user will loose the ability to view all threads from here.
<img width="1164" alt="Capture d’écran 2021-12-01 à 11 52 54" src="https://user-images.githubusercontent.com/8336712/144240120-70292dfe-8346-4287-857c-2679f8de6360.png">

Reply:
Callout only :
<img width="1138" alt="Capture d’écran 2021-12-02 à 11 01 38" src="https://user-images.githubusercontent.com/8336712/144400470-2c1f82f1-9dd0-4690-9865-370b47f58fad.png">

Well only :

<img width="1129" alt="Capture d’écran 2021-12-02 à 11 02 28" src="https://user-images.githubusercontent.com/8336712/144400478-519a89d5-02a7-4fa3-be1d-d7d5173722fb.png">

Reply:
Our green well is more contrasted now.